### PR TITLE
[WIP] Fixing RightSidebar

### PR DESCRIPTION
**Pull request by priyanshu0405**
Added Clear, Refresh and Partymode function to RightSidebar menu.

### CODE REVIEW

**Comment by razzeee** [comment]
This needs a run of elm format, but looks good otherwise

**Comment by priyanshu0405** [comment]
I was wondering how the local works and how to implement it?

File: src/Shared.elm [hunk 1]
```diff
@@ -99,6 +101,7 @@ init flags url key =
       , connection = NotAsked
       , leftSidebarMenuHover = NoneHover
       , rightSidebarExtended = False
+      , isPartyMode = False
```
Comment (razzeee) [inline src/Shared.elm:104]: I think we need to load this on initalization too.

**Comment by razzeee** [comment]
I'm not too sure about that either, other that it seems to do a color switch of the UI.

**Comment by razzeee** [comment]
Merging this, as it's still a clear improvement.

**Pull request merged**